src/main/java/edu/cornell/kfs/fp/CuFPParameterConstants.java

```diff
@@ -50,5 +51,14 @@
         public static final String OBJECT_CODE = "OBJECT_CODE";
         public static final String ACCOUNT = "ACCOUNT";
     }
+    
+    public static class CorporateBilledCorporatePaidDocument {
+        public static final String COMPONENT_NAME = "CorporateBilledCorporatePaidDocument";
+        public static final String DEFAULT_CHART = "DEFAULT_CHART";
```

Comment:
If these are parameter names it would be good for the constants to reflect that.

Reply:
can do